coala_use.md

```diff
@@ -0,0 +1,29 @@
+#Coala
+
+This is a guide on how to use coala-bears for this project
+
+This project consists of a .coafile which basically has all the guidelines
+that needs to be followed when working with coala.It can modified.
+
+In order to specify the files to analyze, you can use the --files argument. 
+For all file paths, you can specify (recursive) globs.
+
+`coala --files=src/\*.c --bears=SpaceConsistencyBear --save`
+
+Coala will now ask you for missing values that are needed to perform the analysis.  
+Coala will now check the code and, in case there are errors it will be duly pointed out.
+You can also run coala in non interactive mode 
+`coala  --non-interactive`  
+
+###Auto-applying results
```

Comment:
Instead of asking user to supply the flags, you can add `default_actions` in the .coafile.

Example: https://github.com/coala/coala-quickstart/blob/master/.coafile#L25 

So adding this default section to the `all` section should do the job for us!